src/ucs/config/parser.c

```diff
@@ -379,6 +398,112 @@ int ucs_config_sprintf_time(char *buf, size_t max, void *src, const void *arg)
     return 1;
 }
 
+int ucs_config_sscanf_bw(const char *buf, void *dest, const void *arg)
+{
+    double *dst     = (double*)dest;
+    char    str[16] = {0};
+    int     offset  = 0;
+    size_t  divider;
+    size_t  units;
+    double  value;
+    int     num_fields;
+
+    num_fields = sscanf(buf, "%lf%16s", &value, str);
+    if (num_fields < 2) {
+        return 0;
+    }
+
+    ucs_assert(num_fields == 2);
+
+    units = (str[0] == 'b') ? 1 : ucs_config_parser_quantity_prefix_value(str[0]);
+    if (!units) {
+        return 0;
+    }
+
+    offset = (units == 1) ? 0 : 1;
```

Comment:
it means something like "10BBps" is also valid?

Reply:
not sure I got question... in case if units==1 then str[0] should be B or b, and next portion of string should be one of ps, /s or s

BBps is not valid

Reply:
right, sorry